src/cmd/factory.go

```diff
@@ -170,6 +170,8 @@ func getEaseeClient() easee.Client {
 			getHTTPClient(),
 			getConfigService(),
 			getConfigService().GetEaseeBaseURL(),
+			20*time.Millisecond,
```

Comment:
20 milisecond command check interval? Also this should be configurable I think. Shouldn't it be like 300 or 500ms?

Reply:
Yeah, true, I'll set it to 300 ms.